RGPosadas/LitterRally
# US-5.2: Showing More Information About Identified Trash

closes #34 

IMPORTANT NOTES:
- Currently my branch is based on `update-ml-model`. I will rebase on develop once #262 is merged.
- Currently, after some investigation with @osahtout @JimLau49, we found out some issues with our implementation regarding iOS. So please test on desktop or an android phone. I'll make a ticket for this.
- I decided to skip on implementing e2e tests for this user story. The main reason is that I can't get chrome to play a video instead of the camera feed and because it would really complicate the e2e process for everyone (it can break on different OSs, etc). I decided it would be better to do manual testing for this feature, since wasting time to possibly get the e2e tests work didn't seem like a good tradeoff
    - There's not that much documentation on this (there's [this page](https://docs.cypress.io/api/plugins/browser-launch-api.html#Use-fake-video-for-webcam-testing) and [this issue](https://github.com/cypress-io/cypress/issues/2704) which were helpful but I still couldn't get it to work).

This PR aims to implement a modal that pops up once you click on a detection icon.

How to test:
- Open the camera page on a camera-able device
- point your camera at an object
- press on the detection icon
- a modal should pop up

![image](https://user-images.githubusercontent.com/23544999/111238484-9173b000-85cd-11eb-806c-f963b0d08b31.png)
![image](https://user-images.githubusercontent.com/23544999/111238491-946ea080-85cd-11eb-8cd7-9adc56272e2c.png)
